Gemfile

```diff
@@ -11,15 +11,15 @@ gem 'webpacker'
 gem 'react-rails'
 
 # Use SCSS for stylesheets
-gem 'sass-rails', '~> 5.0'
+gem 'sass-rails', '~> 6.0'
 
-gem 'bourbon'
-gem 'neat'
+gem 'bourbon', '4.2.6'
+gem 'neat', '1.7.2'
 
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
 # Use CoffeeScript for .coffee assets and views
-gem 'coffee-rails', '~> 4.2.2'
+gem 'coffee-rails' # , '~> 4.2.2'
```

Comment:
Remove this comment or use a specific version.

Reply:
fixed